MAINTAINERS.yml

```diff
@@ -2409,34 +2409,25 @@ Renesas R-Car Platforms:
   status: maintained
   maintainers:
     - aaillet
+    - lorc
```

Comment:
This handle is new to the project, so nothing will be assigned to @lorc etc.  need to go though this first:

https://github.com/zephyrproject-rtos/zephyr/issues/new?assignees=&labels=Role+Nomination&projects=&template=006_nomination.md&title=

Reply:
Done: https://github.com/zephyrproject-rtos/zephyr/issues/61574